README.md

```diff
@@ -228,12 +228,24 @@ libController.on("itemMouseLeave", function (searchText) {
 The string property for the event name is: ItemMouseLeaveEventName. So the following achieves the same:
 
 ```js
-libController.on(libController.ItemMouseLeaveEventName, function(contextData) {
+libController.on(libController.ItemMouseLeaveEventName, function(arg) {
     console.log("Data: " + arg.data);
     console.log("Rect(top, left, bottom, right): " + arg.rect.top + "," + arg.rect.left + "," + arg.rect.bottom + "," + arg.rect.right);
 })
 ```
 
+### Event 'itemSummaryExpanded'
+
+This event is raised when user clicks on the expand icon displayed on the right of a leaf library item or search result item. This event should return data for showing summary, including `InputParameters`, `OutputParameters` and `Description`.
+
+- `arg`: This contains a callback function `onDataReceivedHandler` and `data` of the item.
```

Comment:
Okay after reading the code I understood how this `onDataReceivedHandler` is used. There are two issues:

1. The name is misleading, we need to let client knows it is the way for it to send in the data, so `setDataCallback` (or something similar).
2. The `ReadMe.md` is meant for documenting the event handler's usage, we need a good example here. Something like the following:

```js
libController.on(libController.ItemSummaryExpandedEventName, function (arg) {
  var data = arg.contextData;
  var finalData = {
    "InputParameters": getInputParamsForItem(data),
    "OutputParameters": getOutputParamsForItem(data)
    "Description": getItemDescription(data)
  }

  // Send data back to librarie.js
  arg.setDataCallback(finalData);
});
```

3. `arg.data` should be named `arg.contextData` so that it is less ambiguous.
4. And I do think we need `inputParameters` rather than `InputParameters` as a JSON standard.